app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApiController
+      def index
+        books = Book.all
```

Comment:
If the application has thousands of books, doing something like this could be dangerous.

In general we use some pagination library to avoid the overhead a large collection might generate.
 
I'm not sure if pagination is added in another card, but is something to take into account.